apps/client/src/app/components/user-account-settings/user-account-settings.component.ts

```diff
@@ -42,6 +43,7 @@ export class UserAccountSettingsComponent implements OnDestroy, OnInit {
   public hasPermissionToUpdateViewMode: boolean;
   public hasPermissionToUpdateUserSettings: boolean;
   public isAccessTokenHidden = true;
+  public isFingerprintSupported: boolean;
```

Comment:
What about

`public isFingerprintSupported = this.doesBrowserSupportAuthn();`

and remove the initialization in the constructor?

Reply:
I thought about that. I like it.